cgp/individual.py

```diff
@@ -185,6 +185,8 @@ def __lt__(self, other: "IndividualBase") -> bool:
             assert other_fitness is not None
             if this_fitness < other_fitness:
                 return True
```

Comment:
I am a bit torn on this logic now - it makes sense to sort according to the last hurdle, however it would be kinda cool also to have this customable (eg the sum). At least I think we need to properly document this (in addition to the nice comment of #345). I am making #312 high priority.

Reply:
hmm, not sure what you mean by the sum? added additional documentation to the docstring of `MuPlusLambda`

Reply:
Sorry realised my point made little sense, as this can just be done by the user (had one particular use-case of my self in my mind)